csunplugged/utils/BaseLoader.py

```diff
@@ -58,7 +59,8 @@ def convert_md_file(self, md_file_path, config_file_path, heading_required=True)
         """Return the Verto object for a given Markdown file.
 
         Args:
-            md_file_path: location of Markdown file to convert
+            md_file_path: Location of Markdown file to convert
+            config_file_path: Path to related the config file, used for errors.
```

Comment:
seeing as you are fixing these, you might as well specify that these are strings too